squire_web/src/tournament/overview/mod.rs

```diff
@@ -27,16 +27,32 @@ impl Component for TournOverview {
             .unwrap()
             .state
             .query_tournament(&self.id, |t| {
+                let activeRounds = t.round_reg.rounds.values().filter(|r| r.is_active()).count();
+                let certifiedRounds = t.round_reg.rounds.values().filter(|r| r.is_certified()).count();
+                let registeredPlayers = t.player_reg.players.values().filter(|p| p.can_play()).count();
+                let droppedPlayers = t.player_reg.players.values().filter(|p| !p.can_play()).count();
+
                 html! {
-                    <>
-                    <h1 align="center">{ format!("Welcome to {}", t.name) }</h1>
-                    <p align="center">{ format!("Format : {}", t.format) }</p>
-                    <p align="center">{ format!("Status : {}", t.status) }</p>
-                    <p align="center">{ format!("Number of players : {}", t.player_reg.players.len()) }</p>
-                    <p align="center">{ format!("Number of rounds : {}", t.round_reg.rounds.len()) }</p>
-                    <p align="center">{ format!("Number of judges : {}", t.judges.len()) }</p>
-                    <p align="center">{ format!("Number of admins : {}", t.admins.len()) }</p>
-                    </>
+                    <div class="m-lg-0 m-md-4 m-0">
+                        <div class="p-5 bg-light rounded-3">
+                            <div class="container-fluid p-md-5">
+                                <h1 class="display-5 fw-bold">{ format!("Welcome to {}", t.name) }</h1>
+                                <hr class="my-4"/>
+                                <p>{ format!("Format : {}", t.format) }</p>
+                                <p>{ format!("Status : {}", t.status) }</p>
+                                <p>{ format!("Registered players : {}", registeredPlayers) }</p>
```

Comment:
Nit: you can put `registeredPlayers` in the format string:
```suggestion
                                <p>{ format!("Registered players : {registeredPlayers}") }</p>
```